### PR TITLE
Update Dockerfile for ARM build and the build instruction

### PR DESCRIPTION
**Pull request by NonStatic2014**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -178,9 +178,13 @@ set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)
 5. append "-DCMAKE_TOOLCHAIN_FILE=path/to/tool.cmake" to your cmake args, run cmake and make to build it.
 
 ### Native compiling on Linux (SLOWER)
-Please see [ARM docker file](dockerfiles/Dockerfile.arm32v7). Note that
-to build in ACR-Build (Azure Container Registry), you may want to split it to two files and run them one by one.
-If you run this Dockerfile directly in ACR-Build, it is likely to hit their timeout limitation (8 hours).
+
+Please see [ARM docker file](dockerfiles/Dockerfile.arm32v7). Docker build run on a Raspberry Pi 3B with Raspbian Stretch Lite OS (Desktop version will run out memory when linking the .so file) will take 8-9 hours in total. If you want to use [Azure Container Registry Tasks](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-tasks-overview) building the Docker image in cloud, you may want to split this Dockerfile to two steps:
```
Comment (pranavsharma) [inline BUILD.md]: runs

Comment (pranavsharma) [reply BUILD.md]: building the Docker image -> to build the Docker image.

**Comment by NonStatic2014** [comment]
Is there a way to trigger the CI pipeline again? The failure should not related to this change.

**Comment by pranavsharma** [comment]
> Is there a way to trigger the CI pipeline again? The failure should not related to this change.

done

**Comment by pranavsharma** [comment]
We're making some build pipeline changes. Can you please rebase with the master? Thanks.

File: BUILD.md [hunk 2]
```diff
@@ -178,9 +178,13 @@ set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)
 5. append "-DCMAKE_TOOLCHAIN_FILE=path/to/tool.cmake" to your cmake args, run cmake and make to build it.
 
 ### Native compiling on Linux (SLOWER)
-Please see [ARM docker file](dockerfiles/Dockerfile.arm32v7). Note that
```
Comment (sarobakhtiyari) [inline BUILD.md:181]: Q64SAGPTOV43CLP7

**Review by sarobakhtiyari — APPROVED** [review]

File: onnxruntime/contrib_ops/contrib_ops.cc [hunk 3]
```diff
@@ -12,8 +12,8 @@
 namespace onnxruntime {
 namespace contrib {
 using ::ONNX_NAMESPACE::AttributeProto;
-using ::ONNX_NAMESPACE::OPTIONAL;
```
Comment (sarobakhtiyari) [inline onnxruntime/contrib_ops/contrib_ops.cc:15]: []()ثع

**Review by sarobakhtiyari — APPROVED** [review]

File: onnxruntime/contrib_ops/contrib_ops.cc (hunk 3 above)
Comment (sarobakhtiyari) [reply onnxruntime/contrib_ops/contrib_ops.cc:15]: - - [ ] []()~~~~

Comment (sarobakhtiyari) [reply onnxruntime/contrib_ops/contrib_ops.cc:15]: **__**

**Pull request merged**